test/sandbox/web3/getPastLogs.ts

```diff
@@ -0,0 +1,43 @@
+import assert from 'assert'
+import { web3 } from 'helpers/web3'
+import Logger from 'helpers/Logger'
+
+require('dotenv').config()
+
+/**
+ *  SANDBOX: Get accounts from connected wallet
+ *  RUN:     yarn sandbox test/sandbox/web3/getAccounts.ts
+ */
+const log = new Logger('sandbox:web3:getAccounts')
+
+async function exec (): Promise<void> {
+  const contractAddress = process.env.STABLE_COIN_CONTRACT_ADDRESS
+  assert(contractAddress, 'STABLE_COIN_CONTRACT_ADDRESS is required env var')
+  const topics = [
+    // OrderPlacement
+    // '0x951b66125eb3ef23450a52bb65cf4786c423cc944abb7de8a749a7d05642060e',
+
+    // OrderCancelation
+    // '0x36adb26c9fa7b5219b043ac78b698e6e539a937ff88758ab51b04d331e7111ed'
+
+    // Deposit
+    '0xdcbc1c05240f31ff3ad067ef1ee35ce4997762752e3a095284754544f4c709d7'
```

Comment:
How did you get these topics? `web3.eth.abi.encodeEventSignature`? Should we do that instead, so we don't have these magic numbers here?

Reply:
This is a sandbox, I agree we could do that. But it's testing get past logs in isolation, so you can use any topic.

There's another sandbox to get the topics, you can get it from etherscan too. It's not meant to be "application code" this is why I would find it ok

![image](https://user-images.githubusercontent.com/2352112/71089430-23112980-21a1-11ea-926f-94d004c87bf4.png)

Reply:
cool